.github/ISSUE_TEMPLATE/default-new.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Issue
+about: This template is for a generic new issue.
```

Comment:
*Suggestion:*

Add an `assignees` section for those that will be automatically assigned. Otherwise we run the danger no one is tagged and issues go stale! Perhaps the co-leads for this project team?